packages/dx-react-demos/src/bootstrap3/filtering/remote-filtering.test.jsx

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import Demo from './remote-filtering';
+
+describe('BS3 filtering: remote-filtering', () => {
+  let originalRaf;
+  let originalFetch;
+
+  beforeEach(() => {
+    originalRaf = window.requestAnimationFrame;
```

Comment:
Why do you need to mock `requestAnimationFrame` here?

Reply:
It's a VirtualTable requirement